docs/src/components/power-estimate.js

```diff
@@ -180,6 +186,21 @@ function PowerEstimate({
     });
   }
 
+  if (backlight.backlightEnabled) {
+    const backlightMicroA =
+      backlight.backlightQuantity *
+      backlight.backlightBrightness *
+      backlightPower.ledOn;
+    const backlightMicroW = backlightMicroA * voltageEquivalent;
+    const backlightUsage = backlightMicroW * (1 - usage.percentAsleep);
+
+    totalUsage += backlightUsage;
+    powerUsage.push({
+      title: "Backlight",
+      usage: backlightUsage,
+    });
+  }
```

Comment:
Maybe instead of setting a blanket 20mA per LED, instead the user can input the forward voltage of the LED and the resistors in use? I think this could help quite a bit with accuracy. 

Then the current would be something like `(powerSupply.outputVoltage - backlight.forwardVoltage) / backlight.resistance * backlight.backlightQuantity`. Definitely more work for the user to input, but LEDs can vary quite a bit in power usage. A blue LED hooked up to 3.3V with only 100 ohms of resistance is about 3mA for reference rather than 20mA.

Reply:
Another item we need to consider is the PWM power usage. I can help test to see how much power this uses. IIRC PWM power usage is quite high on nRF52.